br/pkg/storage/writer.go

```diff
@@ -2,11 +2,11 @@ package storage
 
 import (
 	"bytes"
-	"compress/gzip"
 	"context"
 	"io"
 
 	"github.com/golang/snappy"
```

Comment:
Please run ```make bazel_prepare```.

Reply:
addressed in https://github.com/pingcap/tidb/pull/43129/commits/607d490095ddb176d3ec054cf4f9af0a6acc7efa